.travis.yml

```diff
@@ -10,24 +10,30 @@ branches:
 
 jobs:
   include:
+   - stage: check_commits
```

Comment:
This makes the PR build fail if there isn't at least one commit that specifies the type of release with a message like `[major]...`, `[minor]...` or `[bugfix]...`